app/views/bookmarks/_tools.html.erb

```diff
@@ -0,0 +1,13 @@
+<ul class="<%= controller_name %>Tools nav nav-pills">
+  <%= render_show_doc_actions document_list, document: nil, document_list: @document_list, url_opts: Blacklight::Parameters.sanitize(params.to_unsafe_h) do |config, inner| %>
+    <li>
+      <%= inner %>
+    </li>
+  <% end %>
+
+  <li>
```

Comment:
Rather than overriding the partial I think there should be way to configure this to display on the bookmarks page using the Blacklight Document Action conventions: https://github.com/projectblacklight/blacklight/wiki/Adding-new-document-actions